app/controllers/api/v1/reserved_flights_controller.rb

```diff
@@ -0,0 +1,74 @@
+class Api::V1::ReservedFlightsController < ApplicationController
+  before_action :set_reserved_flight, only: %i[show edit update destroy]
+
+  # GET /reserved_flights or /reserved_flights.json
+  def index
+    @reserved_flights = ReservedFlight.all
+    render json: @reserved_flights
```

Comment:
Missing error handling here, what happens if it does not work the way you expect it to?